packages/manager/apps/pci-rancher/src/components/layout-helpers/Dashboard/RancherDetail/RancherDetail.component.tsx

```diff
@@ -119,11 +125,27 @@ const RancherDetail = ({
 
   const onAccessRancherUrl = () =>
     trackAction(TrackingPageView.DetailRancher, TrackingEvent.accessUi);
-
   const shouldDisplayUpdateSoftware =
     getLatestVersionAvailable(rancher, versions) &&
     isReadyStatus &&
     !updateSoftwareResponseType;
+  const locales = useRef({ ...dateFnsLocales }).current;
+  const userLocale = getDateFnsLocale(i18n.language);
+
+  const displayDate = useCallback(
+    (value: string) => {
+      if (userLocale in locales) {
+        const localeId = userLocale as keyof typeof locales;
+        return format(new Date(dateUsage), value, {
+          locale: locales[localeId],
+        });
+      }
+      return format(new Date(dateUsage), value, {
+        locale: locales.fr,
+      });
+    },
+    [userLocale, locales, dateUsage],
+  );
```

Comment:
Maybe we could simplify a bit :)
```suggestion
  const displayDate = useCallback(
    (value: string) =>
      format(new Date(dateUsage), value, {
        locale:
          userLocale in locales
            ? locales[userLocale as keyof typeof locales]
            : locales.fr,
      }),
    [userLocale, locales, dateUsage],
  );
```